projects/packages/videopress/src/client/block-editor/blocks/video/components/poster-panel/index.tsx

```diff
@@ -371,6 +459,64 @@ export default function PosterPanel( {
 		[ attributes ]
 	);
 
+	const onPreviewOnHoverChange = useCallback(
+		( shouldPreviewOnHover: boolean ) => {
+			setPreviewOnHover( shouldPreviewOnHover );
+			let data = {};
+
+			if ( ! shouldPreviewOnHover ) {
```

Comment:
We should not clean these data. It's fine preserving them in case the user enables it again.